packages/DMX/0.1.0/DMX_mac.json

```diff
@@ -0,0 +1,19 @@
+{
+    "arch": "all",
```

Comment:
On mac you'll want to specify this as universal if the end goal is to make a universal binary: https://github.com/ccrma/chugins/blob/main/Line/build-pkg-mac.ck#L36